CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/LoggingRequestHandler.java

```diff
@@ -0,0 +1,88 @@
+package com.couchbase.CouchbaseLiteServ.server.RequestHandler;
+
+import android.content.Context;
+
+import com.couchbase.CouchbaseLiteServ.MainActivity;
+import com.couchbase.CouchbaseLiteServ.server.Args;
+import com.couchbase.lite.Database;
+import com.couchbase.lite.LogFileConfiguration;
+import com.couchbase.lite.LogLevel;
+
+import java.io.File;
+
+public class LoggingRequestHandler {
+    /* ----------- */
+    /* - Logging - */
+    /* ----------- */
+
+    public String configure(Args args){
+        String log_level = args.get("log_level");
+        String directory = args.get("directory");
+        int maxRotateCount = args.get("max_rotate_count");
+        long maxSize = args.get("max_size");
+        boolean plainText = args.get("plain_text");
+
+        if (directory.equals("")) {
```

Comment:
directory == null || directory.isEmpty()

Reply:
directory variable is a string. "" is not equal to null nor string object has isEmpty method, so can't use either.